app/src/main/res/values/strings.xml

```diff
@@ -193,6 +194,7 @@
     <string name="app_settings_distrust_system_certs">Distrust system certificates</string>
     <string name="app_settings_distrust_system_certs_on">System and user-added CAs won\'t be trusted</string>
     <string name="app_settings_distrust_system_certs_off">System and user-added CAs will be trusted (recommended)</string>
+    <string name="app_settings_distrust_system_certs_warning">Are you sure that you want to enable distrusting system certificates? When enabled, you need to manually install the certificates required for your connections, or otherwise they won\'t be possible. This option is targeted to users with high knowledge of networking.</string>
```

Comment:
So I would probably write something else here (see review), but if we want to stick with this I would reword it as: 

"Are you sure you want to enable distrusting system certificates? When enabled, you must manually install the required certificates for your connections; otherwise, they won’t work. This option is intended for users with advanced networking knowledge."